Source/Scene/Gltf3DTileContent.js

```diff
@@ -381,152 +129,67 @@ function initialize(content, gltf) {
     gltf = parseGlb(gltf);
```

Comment:
This `glb` -> `gltf` parsing step is no longer required. The `gltf` object can be passed to `Model` directly which will detect whether it's a glTF or glb.